.changeset/lazy-ducks-lick.md

```diff
@@ -0,0 +1,21 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+Added `LinkBox` and `LinkAction`components to improve components with action accessibility.
+
+Usage:
+```js
+<LinkBox>
+  <LinkAction href='/new-page'>Link to click</LinkAction>
+</LinkBox>
+```
+
+Added `Tile.Action` to take advantage of these new a11y components in Tile.
+
+Usage:
+```js
+<Tile 
+  name={<LinkAction as='button' onClick={() => {/* Do things */}}>Tile name that is also a button</LinkAction>}
+/>
+```
```

Comment:
I think this Tile.Action part should go in a separate changeset, and the example should show Tile.Action instead of LinkAction